airbyte-integrations/connectors/source-monday/source_monday/monday.yaml

```diff
@@ -0,0 +1,90 @@
+version: "0.1.0"
+
+definitions:
+  schema_loader:
+    type: JsonSchema
+    file_path: "./source_monday/schemas/{{ options['name'] }}.json"
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_pointer:
+       - "data"
+       - "{{ options['name'] }}"
+  requester:
+    url_base: "https://api.monday.com/v2"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['credentials']['api_token'] if config['credentials']['auth_type'] == 'api_token' else config['credentials']['access_token'] if config['credentials']['auth_type'] == 'oauth2.0' else config.get('api_token', '') }}"
+    request_options_provider:
+      class_name: "source_monday.GraphQLRequestOptionsProvider"
+      limit: "{{ options['items_per_page'] }}"
+    error_handler:
+      response_filters:
+        - predicate: "{{ 'error_code' in response and response['error_code'] == 'ComplexityException' }}"
+          action: RETRY
+      backoff_strategies:
+        - type: ExponentialBackoffStrategy
+          factor: 30
```

Comment:
why so much?

Reply:
Because of rate limits. I need ask again after 1 minute